course-v2/layouts/partials/course_info_toggle.html

```diff
@@ -0,0 +1,10 @@
+{{ $isCourseHomePage := .isCourseHomePage | default "" }}
+
+{{ if not $isCourseHomePage }}
+  <div class="d-flex justify-content-end">
```

Comment:
**Requested Change:** A button element seems more semantic here than a div-wrapping anchor. 

Same comment below for the menu button (which certainly visually looks like a button).

Reply:
I think this `More Info` is supposed to be a link rather than a button, and wrapping it up with a button will also require some extra styling to hide/remove the button background, border, etc (since button has its own default styles)
But I agree with the `Menu`, it should be a button. 

So I think we can keep `More Info` as it is since it's a link.
and we can replace `div` with `button` for `Menu`.

What do you think?

Reply:
My inclination is that both "Menu" and "More info" perform similar actions (open a drawer) and neither one changes the URL, so they should both be buttons not links. The fact that one is styled differently from the other shouldn't really affect the tag / role. 

As far as styling, you can make a button look like a link with Bootstrap's `btn btn-link` class. https://getbootstrap.com/docs/4.0/components/buttons/#examples  _Another option would be the use `role="button"` on a link, but using an actual button element is probably better if we want a button role._

For reference...[WAI-ARIA says](https://www.w3.org/TR/wai-aria-1.1/#link)
> NOTE
> If pressing the link triggers an action but does not change browser focus or page location, authors are advised to consider using the button role instead of the link role.

Which...isn't really the clearest, imo. In this case the page location (which I think means URL) does not change. I guess maybe browser focus changes? I'm not sure. Anyway, "Does the URL change" seems like a good test to me.

Reply:
I think this makes sense, you're right!
Let's wrap this up with button too

Reply:
Done.
Please have another look